.idea/dictionaries/pmaraitre.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
```

Comment:
Don't worry, you're fast in fixing them :)

Btw, a little tip if you're going to create more PRs: Create a new branch for each change you make. This way, each PR has its own branch and changes in PR 1 aren't included in PR 2.

You can create new branches like this:
```
# move to the 2.8 branch
$ git checkout 2.8
# create a new branch
$ git checkout -b my_awesome_doc_fix
# ... work on the changes
# commit
$ git add path/to/changes.rst
$ git commit -m "I did something great!'
# push to github & create a PR
$ git push ...
```